internal/management/controller/tablespaces/manager.go

```diff
@@ -0,0 +1,60 @@
+package tablespaces
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	apiv1 "github.com/cloudnative-pg/cloudnative-pg/api/v1"
+	"github.com/cloudnative-pg/cloudnative-pg/pkg/management/postgres"
+)
+
+// TablespaceReconciler is a Kubernetes manager.Runnable
```

Comment:
it's not a Runnable. Missing the Start() method, no?

Reply:
It is not a runnable, I put the method to create reconciler into manager.go